apps/i18n/common/en_us.json

```diff
@@ -492,13 +492,15 @@
   "teacherCommunity": "Teacher Community",
   "teacherCommunityDescription": "Ask questions about curriculum, share ideas from your lessons, and get help from other teachers",
   "teacherCourseHoc": "Hour of Code",
-  "teacherCourseHocDescription": "Celebrated in December, but available year-round, the Hour of Code makes computer science fun and accessible to all ages.",
+  "teacherCourseHocDescription": "If you don’t have time for a full length course, try a one-hour tutorial designed for all ages. Join millions of students and teachers in over 180 countries by starting with an Hour of Code.",
```

Comment:
Maybe append "Visit HourOfCode.com to learn more." at the end where the whole thing is hyperlinked? Is that what you mean?